.github/workflows/ContinuousIntegration.yml

```diff
@@ -18,10 +18,10 @@ jobs:
     runs-on: ${{matrix.os}}
     strategy:
       matrix:
-        os: [ubuntu-18.04, macos-10.15]
+        os: [ubuntu-20.04, macos-11]
         #build_type: [ Debug, Release ]
         build_type: [ Release ]
-        vFortran: [ gfortran-9 ]
+        vFortran: [ gfortran-10 ]
 
```

Comment:
standard updates to the VM the tests run on. Ubuntu 18 and macos 10 aren't supported by the runners any longer.